lib/web/servers.go

```diff
@@ -75,9 +76,10 @@ func (h *Handler) clusterDatabasesGet(w http.ResponseWriter, r *http.Request, p
 	}
 
 	return listResourcesGetResponse{
-		Items:      ui.MakeDatabases(h.auth.clusterName, databases),
-		StartKey:   resp.NextKey,
-		TotalCount: resp.TotalCount,
+		Items:                ui.MakeDatabases(h.auth.clusterName, databases),
+		StartKey:             resp.NextKey,
+		TotalCount:           resp.TotalCount,
+		AssumedRoleRequestID: ctx.session.GetAssumedRoleRequestID(),
```

Comment:
i don't think we should be returning the ID with the `listResourcesGetResponse`, it seems out of place. wouldn't this field `AssumedRoleRequestID` have the same value returned for each `listResourcesGetResponse` fetch? 

this value would only change upon renewing session from switching back or assuming roles, which would prompt the browser to refresh to fetch new `userContext` to apply new perms.

could we just store the ID in browser memory with our [get userContext object](https://github.com/gravitational/teleport/blob/master/lib/web/ui/usercontext.go#L101)?